### PR TITLE
E2E: improved readability in ci.yml

### PR DESCRIPTION
**Pull request by GmBodhi**
**Description**

Improves the readability of `CI.yml`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -41,7 +41,9 @@ jobs:
         with:
           node-version: 12
       - name: Install packages
-        run: npm ci && npm ci --prefix test
+        run: |
```
Comment (Mugen87) [inline .github/workflows/ci.yml:44]: Out of curiosity: Why are you using the pipe operator? I'm not that into yaml but I've always thought `>` is the default for multi-line strings.

Comment (GmBodhi) [reply .github/workflows/ci.yml:44]: This is the multi-line operator actually. [stackoverflow](https://stackoverflow.com/questions/15540635/what-is-the-use-of-the-pipe-symbol-in-yaml)

**Comment by GmBodhi** [comment]
I've accidently deleted my fork. Should I re-open this pr? Or will it be fine?

**Comment by Mugen87** [comment]
It still looks good!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**